assembl/tests/views_tests/test_api2.py

```diff
@@ -1180,4 +1186,23 @@ def test_base(self, proposals_with_sentiments, discussion, timeline_phase2_inter
         assert header[self.POST_CLASSIFIER] == b"Classification de Post"
 
     # TODO: Add more unit tests for the phase 2 export API.
-    # TODO: Add unit tests for votes export API.
+
+
+class TestExtractCsvVoters(AbstractExport):
+    view_name = 'extract_csv_voters'
+    NOM_DU_CONTRIBUTEUR = 0
+    NOM_UTILISATEUR_CONTRIBUTEUR = 1
+    EMAIL_DU_CONTRIBUTEUR = 2
+    DATE_HEURE_DU_VOTE = 3
+    PROPOSITION = 4
+
+    def test_base(self, test_app, discussion, token_vote_spec_with_votes, gauge_vote_specification_with_votes, number_gauge_vote_specification_with_votes):
+        result = self.get_result(test_app, discussion.id, gauge_vote_specification_with_votes.widget_id, view_name=self.view_name, votes_export=True)
+        header = result[0]
+        # assert header[self.NOM_DU_CONTRIBUTEUR] == b"Nom Du Contributeur"
```

Comment:
Does this make the test fail? Perhaps uncomment this :)

Reply:
@ayyazdaniaryan  Yes... I should check this with you before the merge.